modules/apps/27-interchain-accounts/controller/keeper/account.go

```diff
@@ -21,13 +21,17 @@ func (k Keeper) RegisterInterchainAccount(ctx sdk.Context, connectionID, owner s
 		return err
 	}
 
-	if k.portKeeper.IsBound(ctx, portID) {
-		return sdkerrors.Wrap(icatypes.ErrPortAlreadyBound, portID)
+	// if there is an active channel for this portID / connectionID return an error
+	activeChannelID, found := k.GetOpenActiveChannel(ctx, connectionID, portID)
```

Comment:
By checking this we're ensuring that an owner address can only have one interchain account per connection. Ideally, an owner could have many accounts per connection. I think this is fine for v1 though. 

There's still nothing stopping ica-auth dev from using owner input parameter however they please though (increment sequence appended or w/e), so it's not a total blocker anyway. 

cc @AdityaSripal @colin-axner @damiannolan